packages/dashboard/src/components/contextMenu/menu.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { usePopper } from 'react-popper';
```

Comment:
Non-blocking: Is this a use case that the Cloudscape popover can't handle? https://cloudscape.design/components/popover/?tabId=playground

Reply:
Hmm I would have to look at all of the features in cloudscape popover. I'm using popperjs for positioning, I don't want to have to contend with any cloudscape styling.